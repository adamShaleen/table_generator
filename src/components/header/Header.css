#header {
    padding: 8px;
    background-color: black;
    color: white;
}
