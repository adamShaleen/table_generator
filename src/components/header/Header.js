import React from 'react';
import './Header.css'

const Header = () => {
    return (
        <div id="header">
            <h4>Table Generator</h4>
        </div>
    )
};

export default Header;
