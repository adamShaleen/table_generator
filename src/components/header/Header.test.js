import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

describe('Snapshot of Header component', () => {
    it('renders correctly', () => {
        const tree = renderer
        .create(<Header/>).toJSON();

        expect(tree).toMatchSnapshot();
    })
})
