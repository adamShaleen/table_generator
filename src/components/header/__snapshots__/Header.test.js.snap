// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of Header component renders correctly 1`] = `
<div
  id="header"
>
  <h4>
    Table Generator
  </h4>
</div>
`;
